packages/mobile/e2e/src/usecases/SetAccountKey.js

```diff
@@ -6,7 +6,7 @@ export default SetAccountKey = () => {
     if (device.getPlatform() === 'android') {
       return
     }
-    await element(by.id('Hamburguer')).tap()
+    await element(by.id('Hamburger')).tap()
```

Comment:
was that the French version of a burguer? :)

Reply:
Changed all instances of 'Hamburguer' to 'Hamburger' for consistency.